docs/experiments-clinical-trials.md

```diff
@@ -0,0 +1,126 @@
+# Pyserini: BM25 and RM3 Baseline for Clinical Trials
+
+This guide contains instructions for running BM25 and RM3 baselines on the [TREC Clinical Trials Track](http://www.trec-cds.org/2021.html).
+
+## Data Prep
+
+The guide requires the [development installation](https://github.com/castorini/pyserini/#development-installation) for additional resource that are not shipped with the Python module; for the (more limited) runs that directly work from the Python module installed via `pip`, see [this guide](pypi-reproduction.md).
+
+We're going to use the repository's root directory as the working directory.
+First, we need to download and extract the Clinical Trials documents and topics.
+
+```
+mkdir collections/trec-ct
+
+wget http://www.trec-cds.org/2021_data/ClinicalTrials.2021-04-27.part1.zip -P collections/trec-ct
+wget http://www.trec-cds.org/2021_data/ClinicalTrials.2021-04-27.part2.zip -P collections/trec-ct
+wget http://www.trec-cds.org/2021_data/ClinicalTrials.2021-04-27.part3.zip -P collections/trec-ct
+wget http://www.trec-cds.org/2021_data/ClinicalTrials.2021-04-27.part4.zip -P collections/trec-ct
+wget http://www.trec-cds.org/2021_data/ClinicalTrials.2021-04-27.part5.zip -P collections/trec-ct
+
+unzip 'collections/trec-ct/*.zip' -d collections/trec-ct
+
+wget http://www.trec-cds.org/topics2021.xml -P tools/topics-and-qrels
+```
+
+Next we need to convert the documents into a json collection.
+
+```
+python scripts/trec-ct/convert_trec21_ct_to_json.py --input_dir collections/trec-ct --output_dir collections/trec-ct-json
+```
+The size of the output json file collections/trec-ct-json/trec21.json should be 2.4G.
+
+Next we convert topics into tsv queries.
+
+```
+python scripts/trec-ct/convert_topic_xml_to_tsv.py --topics tools/topics-and-qrels/topics2021.xml \
+                                                   --queries tools/topics-and-qrels/ctqueries2021.tsv
+```
+
+Build the index with the following command:
+
+```
+python -m pyserini.index -collection JsonCollection \
```

Comment:
change to double hyphens to be consistent with the rest of the system... yes, `--index` now works!